core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -1306,15 +1828,99 @@ public void testGetAllStoreSomeMatchLoaderNoMatchSomeFail() throws Exception {
       fail();
     } catch (BulkCacheLoadingException e) {
       // Expected
-      assertThat(e.getSuccesses(), Matchers.<Map<?,?>>equalTo(union(getEntryMap(KEY_SET_A), getNullEntryMap(KEY_SET_C))));
-      assertThat(e.getFailures().keySet(), Matchers.<Set<?>>equalTo(KEY_SET_D));
+      assertThat(e.getSuccesses(), Matchers.<Map<?,?>>equalTo(union(getEntryMap(KEY_SET_A), getNullEntryMap(KEY_SET_C))));
+      assertThat(e.getFailures().keySet(), Matchers.<Set<?>>equalTo(KEY_SET_D));
+    }
+
+    verify(this.store).bulkComputeIfAbsent(eq(fetchKeys), getAnyIterableFunction());
+    // ResilienceStrategy invoked: no assertion for Store content
+    verify(this.loaderWriter, atLeast(1)).loadAll(this.loadAllCaptor.capture());
+    assertThat(this.getLoadAllArgs(), equalTo(copyWithout(fetchKeys, KEY_SET_A)));
+    verifyZeroInteractions(this.spiedResilienceStrategy);
+
+    validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.GetOutcome.class));
+    validateStats(ehcache, EnumSet.noneOf(CacheOperationOutcomes.CacheLoadingOutcome.class));
+  }
+  
+  /**
+   * Tests {@link Ehcache#getAll(Set)} for
+   * <ul>
+   *    <li>non-empty request key set</li> 
+   *    <li>some {@link Store} entries match</li>
+   *    <li>no {@link CacheLoaderWriter} entries match</li>
+   *    <li>some {@link CacheLoaderWriter#loadAll(Iterable)} calls fail with {@link BulkCacheLoadingException}</li>
+   * </ul>
+   */
+  @Test
+  public void testGetAllStoreSomeMatchLoaderNoMatchSomeFailWithBulkCacheLoadingException() throws Exception {
+    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B));
+    this.store = spy(fakeStore);
+
+    final FakeCacheLoaderWriter fakeLoader = new FakeCacheLoaderWriter(getEntryMap(KEY_SET_F), KEY_SET_C, true);
```

Comment:
Do not start in an inconsistent state: the loader needs to know a super set of what the store knows. If that makes the test a duplicate, delete it.